chpl/chpl-service/src/main/java/gov/healthit/chpl/util/ValidationUtils.java

```diff
@@ -127,7 +128,7 @@ public boolean hasCert(String certNumber, List<CertificationCriterion> allCerts)
         return hasCert;
     }
 
-    public boolean hasCert(CertificationCriterion criterion, List<CertificationCriterion> allCerts) {
+    public boolean hasCriterion(CertificationCriterion criterion, List<CertificationCriterion> allCerts) {
```

Comment:
Was this method not used anywhere else? I can't find anywhere it's being used outside of this file in the PR, except maybe where it was removed from being used. If it _is_ only used inside this class, should it be private? If it's still used somewhere else, were those references changed?

Reply:
It's used in the new reviewers (RequiredAndRelatedCriteriaReviewer.java) . I guess it wasn't used anywhere else before I wrote those classes.